src/indexer/allo/v2/handleEvent.ts

```diff
@@ -1,10 +1,36 @@
 import { EventHandlerArgs } from "chainsauce";
+import { ethers } from "ethers";
 import type { Indexer } from "../../indexer.js";
-import { ProjectTable } from "../../../database/schema.js";
+import { ProjectTable, NewRound } from "../../../database/schema.js";
 import { Changeset } from "../../../database/index.js";
 import { parseAddress } from "../../../address.js";
 import roleGranted from "./roleGranted.js";
 import roleRevoked from "./roleRevoked.js";
+import { fetchPoolMetadata } from "./parsePoolMetadata.js";
+
+function padBytes32Hex(s: string): string {
+  if (s.length > 64) {
+    return s;
+  }
+
+  const padding = 64 - s.length;
+  let hex = s;
+  hex = "0".repeat(padding) + hex;
+  return "0x" + hex;
+}
```

Comment:
We can use https://viem.sh/docs/utilities/pad#pad here

Reply:
@vacekj good catch, I'm going to change it in branch `allo-v2-rounds-4` where I fixed that part converting the poolId to an hex string